demo/app/components/table/table.component.ts

```diff
@@ -108,35 +118,70 @@ export class ExampleHttpDao {
   styleUrls: ['./table.component.scss'],
 })
 export class TableComponent implements AfterViewInit {
-  allColumns = COLUMNS_SOURCE_GITHUB.slice(0);
-  displayedColumns: string[] = [
+  public allColumns = COLUMNS_SOURCE_GITHUB.slice(0);
+  public savedResponse: GithubApi | null = null;;
+  public useCachedData = true;
+  public displayedColumns = [
     'title',
     'updated',
     'comments',
     'assignee',
     'number',
     'labels',
     'created',
-    'id',
     'body',
+    'id',
+    'html_url',
+  ];
+  public exampleDatabase!: ExampleHttpDao;
+  public dataSource = new TsTableDataSource<GithubIssue>();
+  public resultsLength = 0;
+  public resizableColumns: TsColumn[] = [
+    {
+      name: 'title',
+      width: '400px',
+    },
+    {
+      name: 'number',
+    },
+    {
+      name: 'updated',
+      // width: '300px',
+    },
+    // { name: 'comments' },
```

Comment:
Leaving these commented lines in so that when we're developing we can easily change column definitions without rewriting them each time.